src/entities/notification-campaign/notification-campaign.ts

```diff
@@ -219,6 +242,10 @@ export class NotificationCampaign extends UniverseEntity<NotificationCampaignPay
   public armLock?: NotificationCampaignPayload['armLock']
   public statusHistory?: NotificationCampaignPayload['statusHistory']
   public attributionMeta?: NotificationCampaignPayload['attributionMeta']
+  public parent?: NotificationCampaignPayload['parent']
+  public campaignType?: NotificationCampaignPayload['campaignType']
+  public blueprintMeta?: NotificationCampaignPayload['blueprintMeta']
+  public blueprintConfiguration?: NotificationCampaignPayload['blueprintConfiguration']
```

Comment:
This is not used anywhere atm in the sdk. Intended?

Reply:
just to keep the up to date representation of the data that is returned by the api.